common/lib/container-definitions/package.json

```diff
@@ -519,11 +519,6 @@
           "backCompat": false,
           "forwardCompat": false
         }
-      },
-      "0.47.1000": {
```

Comment:
this is probably what's causing the break

Reply:
It breaks locally when this is added back in